LiskHQ/lisk-hub
# Fix UX issues in the sign in page - closes #2412

### What issue have I solved?

-- #2412 

### How have I implemented/fixed it?

This PR is for update the Sign In page and align it with the new design, the following things were update/change.

- [x] Remove splash screen.
- [x] Update all pages where the redirection was pointing to the splash screen.
- [x] Update the Sign In page adding the link for the Login with HWWallet.
- [x] Update title and subtitle in the Sign In page.
- [x] Update passphrase input style (css) to match the new one.

### How has this been tested?

For test all this changes please follow these steps.

1. Run the app.
2. You must see now as default/starting page the Sign In page.
3. Sign In page should looks like the new design.
4. All the functionality should be the same.
5. The link for HWWallet should work and redirect to hw wallet page.

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
